t/apicast-policy-conditional.t

```diff
@@ -0,0 +1,105 @@
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: Conditional policy calls its chain when the condition is true
+In order to test this, we define a conditional policy that only runs the
+phase_logger policy when the request path is /log.
+We know that the policy outputs "running phase: some_phase" for each of the
+phases it runs, so we can use that to verify it was executed.
+--- configuration
+{
+  "services": [
+    {
+      "id": 42,
+      "backend_version":  1,
```

Comment:
We should not need those when APIcast (3scale) policy is not active, right?

Reply:
Right :+1: 
Fixed.